gui/assests/css/dspace-theme.css

```diff
@@ -0,0 +1,352 @@
+/**
```

Comment:
dspace-theme.css and frdr.css can probably be merged into one file.  Likely this one from prod will be close:
https://www.frdr-dfdr.ca/static/css/latest/frdr-theme.css

If it doesn't work then let's take a look at what classes you are using in this app.

Reply:
Merged into one file called frdr-theme.css, and the app looks okay.